plugin-api/src/main/java/org/hyperledger/besu/plugin/data/Quantity.java

```diff
@@ -34,4 +34,11 @@
    * @return The boxed or object based value of the quantity.
    */
   Number getValue();
+
+  /**
+   * Returns a minimal hex string representation of this quantity with all leading zeros truncated.
+   *
+   * @return A minimal hex string representation of this quantity.
+   */
+  String toShortHexString();
```

Comment:
I know we've been using short string many places but it could mean two different things: https://github.com/ethereum/wiki/wiki/JSON-RPC#hex-value-encoding - always as two or no leading zeros (unless it's a zero).  Could we have a more descriptive name such as `quantiyEncodedHexString()` rather than short?

Reply:
This has burned me a couple of times with the retesteth stuff.

Reply:
What about "toMinimalHexString"?

Reply:
This isn't quantity-encoded (there's no special handling for zero values).  It's just a minimal format that consumers can use as a starting point for more specialized formatting.

Reply:
There is special handling for zero, see the org.hyperledger.besu.util.uint.UInt256BytesTest#shortHexStrings test cases.  I specifically added it to handle quantity encoding.

How about `toQuantityHextString`?  I think we need to make it clear we are using the quantity encodeing rules, "short" leads to confusion.